.github/workflows/examples-nrfconnect.yaml

```diff
@@ -192,6 +192,38 @@ jobs:
                     nrfconnect nrf5340dk_nrf5340_cpuapp lighting-app \
                     examples/lighting-app/nrfconnect/build/zephyr/zephyr.elf \
                     /tmp/bloat_reports/
+            - name: Build example nRF Connect SDK Lock App on nRF7002 PDK
```

Comment:
Could we build only all clusters app on nRF70 on PR and the rest only if `if: github.event_name == 'push' || steps.changed_paths.outputs.nrfconnect == 'true'`? Otherwise, it will take lots of time to build all the examples.